history/taskana-simplehistory-rest-spring/src/main/java/pro/taskana/simplehistory/rest/TaskHistoryEventController.java

```diff
@@ -90,7 +91,8 @@
   public TaskHistoryEventController(
       TaskanaEngineConfiguration taskanaEngineConfiguration,
       SimpleHistoryServiceImpl simpleHistoryServiceImpl,
-      TaskHistoryEventRepresentationModelAssembler taskHistoryEventRepresentationModelAssembler) {
+      TaskHistoryEventRepresentationModelAssembler taskHistoryEventRepresentationModelAssembler)
+      throws SQLException {
 
```

Comment:
Because we call here buildTaskanaEngine() from the engineConfiguration, which now - due to the shift of schema creation - throws a SQLException.

Reply:
And why are we building the engine again? We could just autowire the existing engine instead of rebuilding it. @gitgoodjhe can you have a look at this please? Do you agree with me?

Reply:
Basically I agree. Did a quick test and it didn't work instantly. So I would suggest to put this in a separate ticket...maybe in the one that deals with transactionality, which is also long over due?